src/test/java/subway/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,247 @@
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import subway.common.DatabaseCleaner;
+import subway.dto.request.LineRequest;
+
+@DisplayName("지하철 구간 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class SectionAcceptanceTest {
+
+    @LocalServerPort
+    int port;
+
+    @Autowired
+    DatabaseCleaner databaseCleaner;
+
+    Long lineId;
+
+    Map<String, Long> stationMaps = new HashMap<>();
+
+    @BeforeEach
+    void setUp() {
+        RestAssured.port = port;
+        databaseCleaner.execute();
+
+        createFixtureData();
+    }
+
+    void createFixtureData() {
+        //역 생성
+        Long 신논현역 = createStation("신논현역");
+        Long 강남역 = createStation("강남역");
+        Long 양재역 = createStation("양재역");
+
+        //라인 생성(신논현 - 강남)
+        ExtractableResponse<Response> response = LineAcceptanceTest.createLine(LineRequest.builder()
+            .name("신분당선")
+            .color("bg-red-600")
+            .upStationId(신논현역)
+            .downStationId(강남역)
+            .distance(10L)
+            .build());
+
+        lineId = response.jsonPath().getLong("id");
+
+        //구간 추가(강남 - 양재)
+        this.createSection(강남역, 양재역, 10);
+    }
+
+    Long createStation(String name) {
+        Long id = StationAcceptanceTest.createStation(name).jsonPath().getLong("id");
+        stationMaps.put(name, id);
+        return id;
+    }
+
+
+    /**
+     * POST /lines/1/sections
+     *
+     * @return
+     */
+    ExtractableResponse<Response> createSection(Long upStationId, Long downStationId, Integer distance) {
+        Map<String ,String> params = new HashMap<>();
+        params.put("upStationId", upStationId.toString());
+        params.put("downStationId", downStationId.toString());
+        params.put("distance", distance.toString());
+
+        return RestAssured.given().log().all()
+            .body(params)
+            .pathParam("lineId", lineId)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().post("/lines/{lineId}/sections")
+            .then().log().all()
+            .extract();
+    }
+
+    /**
+     * When 지하철 구간을 생성하면
+     * Then 지하철 구간이 생성된다
+     * Then 지하철 라인 역 목록 조회 시 추가 한 구간의 종점 역을 찾을 수 있다.
+     */
+    @DisplayName("지하철 구간을 생성한다.")
+    @Test
+    void createSection() {
+        Long 양재역 = stationMaps.get("양재역");
+        Long 양재시민의숲역 = createStation("양재시민의숲역");
+
+        // when
+        ExtractableResponse<Response> response = this.createSection(양재역, 양재시민의숲역, 10);
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        // then
+        List<String> stations = LineAcceptanceTest.findLine(lineId).jsonPath()
+            .getList("stations.name", String.class);
+        assertThat(stations).containsAnyOf("양재시민의숲역");
```

Comment:
```suggestion
        List<String> stations = LineAcceptanceTest.findLine(lineId).jsonPath()
            .getList("stations.name", String.class);
        assertThat(stations).containsAnyOf("양재시민의숲역");        // then
        var lineResponse = LineAcceptanceTest.findLine(lineId).as(LineResponse.class);
        assertThat(lineResponse.getStations().stream().map(StationResponse::getName)).containsAnyOf("양재시민의숲역");
```
LineResponse에 빈생성자가 있다면 이런 방법도 가능해요

Reply:
이것도 좋은 방법이네요!